src/app/products/details/details.component.ts

```diff
@@ -16,13 +15,19 @@ import { LoaderComponent } from '../../loader/loader.component';
   standalone: true,
   templateUrl: './details.component.html',
   styleUrl: './details.component.scss',
-  imports: [CommonModule, FooterComponent, RouterLink, NavbarComponent,LoaderComponent],
+  imports: [
+    CommonModule,
+    FooterComponent,
+    RouterLink,
+    NavbarComponent,
+    LoaderComponent,
+  ],
 })
-export class DetailsComponent {
-  selectedProduct = this.store.selectSignal(selectedProduct);
-  loading =this.store.selectSignal(isProductsLoading);
-  selectedSize: string = '';
-  id!: string;
+export class DetailsComponent implements OnInit {
+  public selectedProduct = this.store.selectSignal(selectedProduct);
```

Comment:
any element used only in the component should not have the `public` access modifier.

Reply:
The element is used in the html template

Reply:
I think `protected` will still work in the template and is a better `access modifier` than `public`.